htdocs/categories/class/categorie.class.php

```diff
@@ -856,7 +856,7 @@ public function getObjectsInCateg($type, $onlyids = 0, $limit = 0, $offset = 0,
 		$classnameforobj = $this->MAP_OBJ_CLASS[$type];
 		$obj = new $classnameforobj($this->db);
 
-		$sql = "SELECT c.fk_".(empty($this->MAP_CAT_FK[$type]) ? $type : $this->MAP_CAT_FK[$type]);
+		$sql = "SELECT c.fk_".(empty($this->MAP_CAT_FK[$type]) ? $type : $this->MAP_CAT_FK[$type]).", o.*";
```

Comment:
Usage of * for fields is forbidden.
Please use explicit column name

Reply:
As there is no real standard between all type of objetc that can be categorised to define the status of the object, I 've used o.* to avoir SQL error. 
For the moment, I've found "statut", " "status" and "clos" in dôme object, but I have renonced trying to map all table, since this function may be used by external module...